app/src/main/cpp/skyline/kernel/types/KProcess.cpp

```diff
@@ -28,6 +29,7 @@ namespace skyline::kernel::type {
     void KProcess::Kill(bool join, bool all, bool disableCreation) {
         Logger::Warn("Killing {}{}KProcess{}", join ? "and joining " : "", all ? "all threads in " : "HOS-1 in ", disableCreation ? " with new thread creation disabled" : "");
         Logger::EmulationContext.Flush();
+        state.jvm->reportCrash();
```

Comment:
`Kill` is also called when the game hasn't crashed and we want to quit the game gracefully, showing a crash dialog there wouldn't make sense. You can workaround that by using the `disableCreation` parameter for now, as that is `true` only when gracefully exiting.

```suggestion
        // disableCreation is set only when gracefully exiting, it being false means an exception/crash occurred
        if (!disableCreation)
            state.jvm->reportCrash();
```